apps/passport-server/src/services/generic-issuance/subservices/PipelineExecutorSubservice.ts

```diff
@@ -340,6 +345,12 @@ export class PipelineExecutorSubservice {
       "s from now"
     );
 
+    await Promise.allSettled(
+      this.pipelineSlots
+        .slice()
+        .map((s) => this.restartPipeline(s.definition.id, true))
```

Comment:
So earlier in this function we load all of the pipelines, and here we restart them, but without performing a load (the second parameter to restartPipeline being `true` means that no load is performed). This reads a bit oddly to me, perhaps because the difference between (re)starting and loading a pipeline isn't immediately obvious. I think the side-effect of restarting the pipeline is that it refreshes the definition from the DB, and that's why we're doing it? If so a comment to that effect would be helpful, as it's not immediately obvious that this is why it's being done.